source/StellarObject.h

```diff
@@ -71,6 +72,10 @@ class StellarObject : public Body {
 	// Find out how far this object is from its parent.
 	double Distance() const;
 
+	// Get this object's icon (to be display on the map).
```

Comment:
```suggestion
	// Get this object's icon (to be displayed on the map).
```

Reply:
as we discussed in discord I'll remove the icons so this means this comment is obselete